bot/app/database/db_update.py

```diff
@@ -184,57 +66,13 @@ def insert_plays_at(self):
         """)
     
         if rel_type == "E":
-            ev = Event()
+            ev = new_event()
         elif rel_type == "A":
-            ar = Artist()
+            ar = new_artist()
         elif rel_type == "P":
-            pa = playsAt()
+            pa = plays_at_relation()
         else:
             print("Program is shut down")
```

Comment:
I would just call `break` here and not print anything (since you're also not printing when leaving the `proceed` loop)